CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
 - Fixes an issue where `emulators:exec` would return a 0 error code when emulators failed to start. (#7974)
 - Added `--import` and `emulators:export` support to the Data Connect emulator.
 - Added `firebase.json#emulators.dataconnect.dataDir`. When set, Data Connect data will be persisted to the configured directory between emulator runs.
+- Added App Hosting Emulator
```

Comment:
If there are docs, might be a good idea to include a link to them here

Reply:
docs are in flight

Reply:
added the link that we expect it to be